pkg/controller/submariner/submariner_controller.go

```diff
@@ -537,7 +537,7 @@ func newGlobalnetDaemonSet(cr *submopv1a1.Submariner) *appsv1.DaemonSet {
 							Env: []corev1.EnvVar{
 								{Name: "SUBMARINER_NAMESPACE", Value: cr.Spec.Namespace},
 								{Name: "SUBMARINER_CLUSTERID", Value: cr.Spec.ClusterID},
-								{Name: "SUBMARINER_EXCLUDENS", Value: "submariner,kube-system,operators"},
+								{Name: "SUBMARINER_EXCLUDENS", Value: "submariner-operator,kube-system,operators"},
```

Comment:
If this is wrong, there are at least two other places it seems it should be changed.

```
submariner-operator/pkg/controller/submariner/submariner_controller.go:534						
submariner-operator/pkg/controller/servicediscovery/servicediscovery_controller.go:207							
submariner-charts/submariner/templates/globalnet.yaml:33
```

Reply:
Yes @dfarrell07 I noticed that but I'm not sure if Service Discovery/LH controller also needs some grace period. Basically, gracePeriod is required if a Pod wants to do cleanup before its terminated. 

If the other components also require cleanup, IMO we can propose a separate PR for that. WDYS?

Reply:
And if this is wrong, can we explain why more in the commit message? Might also be better in a separate commit from the timing change.

Reply:
I mean the SUBMARINER_EXCLUDENS change has parallels elsewhere, not the grace period. Can we explain why that's wrong in the commit message (and/or maybe use two commits)?

Reply:
Ack. Originally this PR was focused only on Globalnet Pod and the scope has increased based on the review comments. I added a brief note on the EXCLUDENS change, but I agree this can be a separate commit as well. Let me update it. BTW the EXCLUDENS is not a fix, its more like an optimization. I'll elaborate more in the commit message.

Reply:
Okay, thanks @sridhargaddam!

Reply:
thanks for reviewing @dfarrell07 :)